virtual-organizations/IFAHURP-PRP.yaml

```
@@ -0,0 +1,34 @@
+AppDescription: Astronomy and Astrophysics big data from various surveys, simulations and inversions.
+Community: The Institute for Astronomy, (University of Hawaii, Manoa) conducts fundamental astrophysical research, operates astronomical telescopic surveys and develops instrumentation with many collaborators locally, nationally and globally.
+CertificateOnly: false
+Community: Information Technology Center at the University of Hawaii, Manoa
+Contacts:
+  Administrative Contact:
+  - ID: ef5d0be3ec1df913935853ed82396b1eb37a7db0
+    Name: Fabio Andrijauskas
+Disable: false
+PrimaryFields:
+  - Astronomy
+  - Astrophysics
+SecondaryFields:
+  - Wide field astronomical surveys
+  - Solar Spectropolarimetry
+  - Time domain astronomy
+ID: 10341
+LongName: University of Hawaii Information Technology Center
+OASIS:
+  Managers:
+    - Name: Fabio Andrijauskas
+      DNs: /DC=org/DC=cilogon/C=US/O=University of California-San Diego/CN=Fabio Andrijauskas A51485301
+      ID: ef5d0be3ec1df913935853ed82396b1eb37a7db0
+  UseOASIS: true
+DataFederations:
+  StashCache:
+    Namespaces:
+      /uhawaii/manoa/ifa
+        - PUBLIC
+      /uhawaii/manoa/ifa
+    AllowedOrigins:
+      - IFAHURP_OSDF_ORIGIN
+    AllowedCaches:
+      - ANY
```

```suggestion
    Namespaces:
      - Path: /uhawaii/manoa/ifa
        Authorizations:
          - PUBLIC
        AllowedOrigins:
          - IFAHURP_OSDF_ORIGIN
        AllowedCaches:
          - ANY
```
This is the new namespace schema I am documenting in https://github.com/osg-htc/docs/pull/17